README.md

```diff
@@ -35,9 +35,12 @@ REQUIRED SECTIONS:
 - [📝 License](#license)
 
 ### PROJECT NAME
->Set up and mobile first
 
-> Set up and mobile first was intended to make portfolio
+>FINISH MOBILE VERSION
+
+>finish mobile version
+
+
 
 
 ### It is built by 
```

Comment:
- [OPTIONAL] I think it would be better if you document and style your README according to the [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided 📖. Currently, your README's **_Table of content_** is good but few other content is not documented according to template provided. The README is also missing a **_Getting started_** header. I know you can do that 😃💪
- [ ] Please note that I am pointing this part but please apply it to your whole README 📝